Week4/BlockbusterRecycler/app/src/main/java/com/e/blockbusterrecycler/ModelMovies.kt

```diff
@@ -0,0 +1,51 @@
+package com.e.blockbusterrecycler
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class ModelMovies (
+    val id: Int,
+    val releaseDate: String,
+    val title: String,
+    val summary: Int,
+    val poster:Int,
+    val detail:Int,
+    val director: String,
+    val stars: Int
+): Parcelable {
+    constructor(parcel: Parcel) : this(
+        parcel.readInt(),
+        parcel.readString()!!,
+        parcel.readString()!!,
+        parcel.readInt(),
+        parcel.readInt(),
+        parcel.readInt(),
+        parcel.readString()!!,
```

Comment:
If possible, it's best to use one of the ? operators instead of the forced unwrapping as forced unwrapping can crash.